docs-requirements.in

```diff
@@ -2,7 +2,7 @@
 # sphinx 5.3 doesn't work with our _NoValue workaround
 sphinx >= 6.0
 jinja2
-sphinx_rtd_theme
+sphinx_rtd_theme >= 2
```

Comment:
could add a short comment explaining the `>=` requirement

Reply:
Yep, it's just cause how `uv` locks things, can definitely leave a comment.